cmd/kubeless/ingress.go

```diff
@@ -31,7 +31,7 @@ func init() {
 
```

Comment:
`routeCreateCmd, routeListCmd, routeDeleteCmd`

Reply:
(I guess that the file names should change as well)

Reply:
I am 0 with this. It's fine to change to route 👌